.github/workflows/rust-tests.yaml

```diff
@@ -29,8 +33,8 @@ jobs:
 
       - uses: Swatinem/rust-cache@v2
 
-      # Build Emily first so that the autogenerated `lib.rs` file is
-      # present when building the rest of the crates in the workspace.
-      - run: cargo build --package emily
+      - name: Install package dependencies.
+        run: make install
 
-      - run: cargo test
+      - name: Run tests
+        run: make test
```

Comment:
Should we just update this Github workflow to be something like "Run tests", since this will run both rust and JS tests? I'd be fine with that, and if there are Rust / Contract / etc workflows we want separately we can do that.

Reply:
Yeah, I'll change this to just be `run tests` and keep the contracts one because it looks like it does more than just this does. Ideally these wouldn't need to be separate because GitHub workflowss aren't really for testing, they're for preventing regression, and so long as the workflow is blocked when we mess something and we cursorily know why then we're good.